keep/api/config.py

```diff
@@ -20,6 +21,7 @@ def on_starting(server=None):
 
     migrate_db()
     launch_uptime_reporting()
+    launch_demo_mode()
```

Comment:
wdyt if
```suggestion
    if LIVE_DEMO_MODE:
        launch_demo_mode()
```
?

Reply:
up to you, wanted to manage the env var within the module :)

Reply:
again, it's just me being extra cautious with these things we plug in that are not "production". I would even go further with:
```suggestion
    if LIVE_DEMO_MODE:
        from keep.api.core.demo_mode_runner import launch_demo_mode
        launch_demo_mode()
```
but your call on this one